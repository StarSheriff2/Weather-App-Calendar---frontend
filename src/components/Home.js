import React from 'react';
import { Redirect } from 'react-router-dom';
import { useSelector } from 'react-redux';

const Home = () => {
  const { user: currentUser } = useSelector((state) => state.auth);

  if (!currentUser) {
    return <Redirect to="/" />;
  }

  return (
    <div className="container">
      <header className="jumbotron">
        <h3>
          <strong>Current Reminders</strong>
        </h3>
      </header>
      <p>...Under Construction</p>
    </div>
  );
};

export default Home;
