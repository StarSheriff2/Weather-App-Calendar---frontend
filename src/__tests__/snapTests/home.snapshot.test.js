import React from 'react';
import { cleanup } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderWithRedux from '../../tests/test-utils';
import Home from '../../components/Home';

afterEach(cleanup);

describe('Home', () => {
  test('Renders the Component', () => {
    const home = renderWithRedux(
      <Router>
        <Home />
      </Router>,
    ).toJSON();

    expect(home).toMatchSnapshot();
  });
});
