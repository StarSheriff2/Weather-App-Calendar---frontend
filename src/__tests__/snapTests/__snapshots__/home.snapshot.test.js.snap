// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Renders the Component 1`] = `null`;
